internal/encryption/base.go

```diff
@@ -36,14 +36,14 @@ func newBaseEncryption(enc *encryption, target *config.TargetConfig, enforced bo
 	}
 	// This performs a e2e validation run of the config -> methods flow.  It serves as a validation step and allows us to
 	// return detailed diagnostics here and simple errors below
-	_, diags := base.buildTargetMethods(make(map[keyprovider.Addr]any))
+	_, diags := base.buildTargetMethods(make(map[keyprovider.Addr][]byte))
 	return base, diags
 }
 
 type basedata struct {
-	Meta    map[keyprovider.Addr]any `json:"meta"`
-	Data    []byte                   `json:"encrypted_data"`
-	Version string                   `json:"encryption_version"` // This is both a sigil for a valid encrypted payload and a future compatability field
+	Meta    map[keyprovider.Addr][]byte `json:"meta"`
```

Comment:
I feel like I've already seen the move in the opposite direction in a previous PR.

Would appreciate to hear the reasoning behind it.